recipes/panaroo/meta.yaml

```diff
@@ -19,12 +19,19 @@ build:
     - panaroo-spydrpick = panaroo.spydrpick:main
     - panaroo-img = panaroo.estimate_img_params:main
     - panaroo-fmg = panaroo.estimate_many_genes_params:main
+    - panaroo-msa = panaroo.post_run_alignment_gen:main
 requirements:
   host:
     - python >=3
     - pip
+    - zlib
+    - openmp 
+    - llvm-openmp  # [osx]
```

Comment:
Is there a reason these are needed? This is listed as a pure-python package and zlib/openmp/llvm-openmp are only needed if there's compiled code.

Reply:
I found that the cd-hit dependency installs a single threaded version by default in some cases if these are not included.